examples/mobile/HomeApp/js/app.js

```diff
@@ -349,17 +349,38 @@ import Storage from "./clipping-storage.js";
 	}
 
 	function updateWebClipsUI() {
-		var current = document.querySelectorAll(".ui-card"),
-			webclipsContainer = document.getElementById("web-clips");
+		var webclipsContainer = document.getElementById("web-clips"),
+			currentWebClipsCards = [].slice.call(webclipsContainer.querySelectorAll(".ui-card[data-url],.ui-card[data-src]")),
```

Comment:
[].slice.call is not obvious,
please add some comment about this code here.

Reply:
Done